authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,83 @@
+# Call Microsoft Graph from a Cloud Solution Provider application
+
+> **Note:** This topic applies **only** to Microsoft Cloud Solution Provider (CSP) application developers. The <a href="https://partner.microsoft.com/en-US/cloud-solution-provider" target="_newtab">Microsoft Cloud Solution Provider (CSP)</a> program enables Microsoft’s partners to resell and manage Microsoft Online services to customers. 
+
+This topic describes how to enable application access to partner-managed customer data via Microsoft Graph using either the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a>. 
```

Comment:
Dan, is there a reason you specify link in HTML syntax instead of markdown syntax? This will be a topic in the MS Graph doc set, right? MD Scanner will report these HTML links as broken (I just checked with Ryan).
Also, if you use MD syntax links, because your links go to another site, those destination pages will be opened in a separate tab anyway. Just an FYI in case that was your reason for using HTML syntax links.